### PR TITLE
Fix not necessary url_escape in cookie

### PR DESCRIPTION
**Pull request by norguhtar**
http server add uri_escape to cookie set. But it not need, and broke cookie

### CODE REVIEW

File: rockspecs/http-1.0.1-1.rockspec [hunk 1]
```diff
@@ -1,12 +1,13 @@
 package = 'http'
 version = '1.0.1-1'
 source  = {
-    url = 'git://github.com/tarantool/http.git',
-    tag = '1.0.1',
+    url = 'git://github.com/get-net/http.git',
```
Comment (ASverdlov) [inline rockspecs/http-1.0.1-1.rockspec]: I guess you forgot to delete these changes?

File: rockspecs/http-1.0.2-1.rockspec [hunk 2]
```diff
@@ -1,12 +1,13 @@
 package = 'http'
 version = '1.0.2-1'
 source  = {
-    url = 'git://github.com/tarantool/http.git',
```
Comment (ASverdlov) [inline rockspecs/http-1.0.2-1.rockspec]: I guess you forgot to delete these changes?

File: rockspecs/http-scm-1.rockspec [hunk 3]
```diff
@@ -1,12 +1,13 @@
 package = 'http'
 version = 'scm-1'
 source  = {
-    url    = 'git://github.com/tarantool/http.git',
-    branch = 'master',
+    url    = 'git://github.com/get-net/http.git',
```
Comment (ASverdlov) [inline rockspecs/http-scm-1.rockspec]: I guess you forgot to delete these changes?

File: http/server.lua [hunk 4]
```diff
@@ -635,7 +635,11 @@ end
 
 local function handler(self, request)
     if self.hooks.before_dispatch ~= nil then
-        self.hooks.before_dispatch(self, request)
+        local resp = self.hooks.before_dispatch(self, request)
```
[the remark this answers is not in the document]
Comment (norguhtar) [reply http/server.lua]: It not need. I clean that.

[the remark this answers is not in the document]
Comment (norguhtar) [reply http/server.lua]: Yes )

**Comment by norguhtar** [comment]
Remove and fix not need changes in last commit on PR

**Pull request merged**